package.json

```diff
@@ -35,22 +46,7 @@
     "@types/uuid": "^3.4.3",
     "@types/vinyl": "^2.0.0",
     "@types/vinyl-fs": "^2.4.8",
-    "babel-core": "^6.26.0",
-    "babel-plugin-external-helpers": "^6.22.0",
-    "babel-plugin-syntax-async-functions": "^6.13.0",
-    "babel-plugin-syntax-async-generators": "^6.13.0",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "babel-plugin-syntax-exponentiation-operator": "^6.13.0",
-    "babel-plugin-syntax-export-extensions": "^6.13.0",
-    "babel-plugin-syntax-object-rest-spread": "^6.13.0",
-    "babel-plugin-transform-async-generator-functions": "^6.24.1",
-    "babel-plugin-transform-async-to-generator": "^6.24.1",
-    "babel-plugin-transform-es2015-modules-amd": "^6.24.1",
-    "babel-plugin-transform-exponentiation-operator": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-es2015": "^6.24.1",
-    "babel-preset-minify": "^0.3.0",
-    "babel-traverse": "^6.26.0",
+    "babel-preset-minify": "^0.4.0-alpha.caaefb4c",
```

Comment:
I think we should pin to that version of preset-minify, as ^ isn't likely to do what we want when ordering SHAs like `caaefb4c`

Reply:
done